docs/rich-text/converting-html.mdx

```diff
@@ -6,14 +6,14 @@ desc: Converting between lexical richtext and HTML
 keywords: lexical, richtext, html
 ---
 
-## Converting Rich Text to HTML
+## Rich Text to HTML
```

Comment:
We should update the other converter docs as well then (jsx, mdx etc.). Tried to keep these headings consistent